src/handlers/StatesBtn.ts

```
@@ -6,17 +6,20 @@ import { database } from "../utils/base"
 import { basicColor } from "../utils/consts"
 import { FailedStatusComponent, SuccessStatusComponent } from "../utils/embeds"
 import { DeleteCurrentState, LoadState, SaveState, StatesId } from "../utils/interactionIds"
-import { blacklistUsers, rerenderDashboard } from "../utils/misc"
+import { blacklistUsers, getLocale, rerenderDashboard } from "../utils/misc"
 import { ChannelNotFound, CheckRights } from "../utils/preconditions"
+import { Dict, getT } from "../generated/i18n"
```

ESLint

[error] Imports "Dict" are only used as type.

(`@typescript-eslint/consistent-type-imports`)

---

[error] Unable to resolve path to module '../generated/i18n'.

(import-x/no-unresolved)